### PR TITLE
Self service/create ec2 instance

### PR DESCRIPTION
**Pull request by mk-armah**
# Description

Moving [Create EC2 Example](https://github.com/port-labs/self-service-actions-examples/blob/main/github-workflows/aws/README.md#create-an-aws-ec2-instance-action) Github action to port docs 

- Resource Path (/create-self-service-experiences/setup-backend/github-workflow/examples/create-an-ec2-instance)

### CODE REVIEW

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/create-an-ec2-instance.md [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+
+# Create An AWS EC2 Instance
+
+import PortTooltip from "/src/components/tooltip/tooltip.jsx";
+
+In the following guide, you are going to create a self-service action in Port that executes a [GitHub workflow](/create-self-service-experiences/setup-backend/github-workflow/github-workflow.md) to create an EC2 Instance in AWS using Terraform templates.
+
+## Prerequisites
+
+1. Create the following GitHub Action secrets:
```
Comment (hadar-co) [inline docs/create-self-service-experiences/setup-backend/github-workflow/examples/create-an-ec2-instance.md:13]: @mk-armah Please dont use numbered lists inside other numbered lists, it looks confusing. Consider using bullets where needed.

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/create-an-ec2-instance.md [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+    "required": []
+  },
+  "mirrorProperties": {},
+  "calculationProperties": {},
+  "relations": {}
+}
+```
+</details>
+
+2. Create a Port action in the [self-service page](https://app.getport.io/self-serve) or with the following JSON definition for the `EC2 Instance` blueprint:
```
Comment (hadar-co) [inline docs/create-self-service-experiences/setup-backend/github-workflow/examples/create-an-ec2-instance.md:103]: should be 3?

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/create-an-ec2-instance.md [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+  },
+  "trigger": "CREATE",
+  "description": "Create An EC2 Instance from Port",
+  "requiredApproval": false
+}
+]
+```
+</details>
+
+3. Create a workflow file under .github/workflows/create-an-ec2-instance.yaml with the following content:
```
Comment (hadar-co) [inline docs/create-self-service-experiences/setup-backend/github-workflow/examples/create-an-ec2-instance.md:174]: 4?

**Review by hadar-co — CHANGES_REQUESTED** [review]

**Review by hadar-co — APPROVED** [review]

**Comment by mk-armah** [comment]
@hadar-co , i'm not seeing the changes on port docs yet. Everything okay ?

**Pull request merged**